samples/excel/89-preview-apis/style.yaml

```diff
@@ -0,0 +1,241 @@
+                styles.getItem("Diagonal Orientation Style").orientation = 38;
+                styles.getItem("Diagonal Orientation Style").addIndent = true;
+               
+                let newStyle = styles.getItem("Diagonal Orientation Style");
+                context.load(newStyle);
+
+                newStyle.includeProtection = true;
+                newStyle.shrinkToFit = true;
+                newStyle.locked = false;
+
+                await context.sync();
+
+                OfficeHelpers.UI.notify("Successfully added a new style with diagonal orientation to the Home tab ribbon.");
+            });
+        }
+
+        async function applyNewStyle() {
+            await Excel.run(async (context) => {
+                let worksheet = context.workbook.worksheets.getItem("Sample");
+                context.load(worksheet);
+
+                // Apply new style.
+                worksheet.getRange("A1:E1").style = ("Diagonal Orientation Style");
+                worksheet.getRange("A1:E1").format.verticalAlignment = "Justify";
+
+                let range = worksheet.getRange("A1:E1");
+                context.load(range);
+                context.load(range.format);
+
+                await context.sync();
+            });
+        }
+
+        async function applyBuiltInStyle() {
+            await Excel.run(async (context) => {
+                let worksheet = context.workbook.worksheets.getItem("Sample");
+                context.load(worksheet);
+
+                // Apply built-in style. 
+                // Styles are in the Home tab ribbon.
+                worksheet.getRange("A1:E1").style = Excel.BuiltInStyle.neutral;
+                worksheet.getRange("A1:E1").format.horizontalAlignment = "Right";
+
+                let range = worksheet.getRange("A1:E1");
+                context.load(range);
+                context.load(range.format);
+
+                await context.sync();
+            });
+        }
+
+        async function getStyleFontProperties() {
+            await Excel.run(async (context) => {
+                let style = context.workbook.styles.getItem("Normal");
+                context.load(style);
+                style.font.load("bold, color, italic, name, size");
+                style.fill.load("color");
+
+                await context.sync();
+
+                console.log("Bold: " + style.font.bold);
+                console.log("Font color: " + style.font.color);
+                console.log("Italic: " + style.font.italic);
+                console.log("Name: " + style.font.name);
+                console.log("Size: " + style.font.size);
+                console.log("Fill color: " + style.fill.color);
+            });
+        }
+
+        async function getStyleAlignmentProperties() {
+            await Excel.run(async (context) => {
+                let style = context.workbook.styles.getItem("Diagonal Orientation Style");            
+                context.load(style);
```

Comment:
Note that context.load(someobject) is just as much an empty load as someobject.load(); 
It is a good practice to avoid empty loads. It would be better to have:
style.load('orientation, horizontalAlignment, ... and so on );